packages/amplify-e2e-tests/src/__tests__/delete.test.ts

```diff
@@ -72,25 +72,25 @@ describe('amplify delete', () => {
     await testDeletion(projRoot, { android: true });
   });
 
-  it('should not delete amplify app', async () => {
-    const projRoot2 = await createNewProjectDir('delete-dep');
-    const envName = 'testdelete';
-    await initJSProjectWithProfile(projRoot, {});
-    await addApiWithoutSchema(projRoot);
-    const amplifyMeta = getProjectMeta(projRoot);
-    const meta = amplifyMeta.providers.awscloudformation;
-    const { AmplifyAppId, Region, StackName, DeploymentBucketName } = meta;
-    expect(AmplifyAppId).toBeDefined();
-    await createEnv(AmplifyAppId, envName, Region, StackName, DeploymentBucketName);
-    await pullProject(projRoot2, { appId: AmplifyAppId, envName });
-    await initIosProjectWithProfile(projRoot2, {});
-    await deleteProject(projRoot);
-    expect(await appExists(AmplifyAppId, Region)).toBeTruthy();
-    // clean up
-    await deleteProject(projRoot2);
-    deleteProjectDir(projRoot2);
-    await deleteAmplifyApp(AmplifyAppId, Region);
-  });
+  // it('should not delete amplify app', async () => {
```

Comment:
Add some comments on why this test is removed, for future reference and it would be better to use ```it.skip``` instead of commenting out.